core/src/mmu.cpp

```diff
@@ -6,9 +6,7 @@
 
 namespace n_e_s::core {
 
-uint8_t Mmu::read_byte(uint16_t addr) const {
-    return *translate_address(addr);
-}
+uint8_t Mmu::read_byte(uint16_t addr) const { return *translate_address(addr); }
```

Comment:
This is a bit annoying. What about changing "AllowShortFunctionsOnASingleLine" to SFS_Empty or SFS_Inline?

Reply:
I agree. I completely missed that when messing around with the `.clang-format`. I've pushed an updated version that's more reasonable with fewer things on a single line.